raiden/tests/integration/test_provisioning.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+import pytest
+import gevent
+from gevent.pool import Pool
+from ethereum import slogging
+
+from raiden.tests.utils.blockchain import wait_until_block
+
+log = slogging.getLogger(__name__)
+
+
+@pytest.mark.parametrize('number_of_nodes', [6])
+@pytest.mark.parametrize('channels_per_node', [0])
+@pytest.mark.parametrize('cached_genesis', [False])
+@pytest.mark.parametrize('register_tokens', [True, False])
+def test_participant_selection(
+    raiden_network,
+    tokens_addresses,
+    settle_timeout,
+    reveal_timeout,
+    blockchain_type
+):
+    token_address = tokens_addresses[0]
```

Comment:
I am a bit confused by the name `tokens_addresses`. Should it not just be `token_addresses`?